README.md

```diff
@@ -30,7 +30,19 @@ We need many different skills and areas of expertise to build this tool, and you
 
 ## About the CRISPR system
 
-Interestingly, a mechanism for “adaptive” immune system in bacteria against the viruses infecting them (usually referred to as phages) was discovered just a few years ago, called CRISPR-Cas. Curiously, much like the anti-virus software and intrusion detection systems that rely on detecting patterns found in malicious code, bacteria keeps a dynamic library of small pieces of phage genomes (spacers) to detect and neutralize phage attacks.  
+**C**lustered **R**egularly **I**nter-spaced **S**hort **P**alindromic **R**epeats (CRISPR), and associated proteins (Cas) are part of the CRISPR-Cas system in bacteria. First observed in 1987 (Ishino et al., 1987), the CRISPR system acts like the immune system for bacteria.
```

Comment:
"Interspaced" should be without the dash, I believe. The last phrase "the CRISPR system acts like the immune system for bacteria" could be shortened to just "the CRISPR system is an adaptive immune system for bacteria". 

Also, I like the bolded first letters of the acronym, great idea!